test/k8sT/manifests/test-cli.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: foo
+  labels:
+    id: foo
+spec:
+  containers:
+  - name: netperf
+    image: cilium/netperf
+  nodeSelector:
+    "cilium.io/ci-node": k8s1
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: bar
+  labels:
+    id: bar
+spec:
+  containers:
+  - name: netperf
+    image: cilium/netperf
```

Comment:
`docker.io/cilium/netperf`

Reply:
Fixed. We should really fix this across the codebase, I keep copying it around :worried: